cmd/action/apply.go

```diff
@@ -18,12 +18,12 @@ import (
 var (
 	// ApplyFlags are the flags used in Apply command.
 	ApplyFlags struct {
-		DSN    string
-		File   string
-		Web    bool
-		Addr   string
-		DryRun bool
-		Schema []string
+		DSN         string
+		File        string
+		Web         bool
+		Addr        string
+		DryRun      bool
+		Schema      []string
```

Comment:
is our formatting different?

Reply:
No. One of the users added the auto-approve flag and apparently did not run gofmt. My IDE does this on every commit. And this is what it spit out.

Reply:
Weird though, that the linter does not catch it :thinking: